src/server/game-instance.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+
+const redux = require('redux')
+
+exports.findGameData = (gameId) => {
+  if (gameId == 'tic-tac-toe') {

```

Comment:
We probably want to also type check here, because it's an id (and an id could be anything unique)